app/src/main/res/drawable/ic_code_of_conduct.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="24dp"
-    android:height="24dp"
+    android:width="50dp"
```

Comment:
Don't hardcode dimens value in the xml file. Write them int the dimens file and use them everywhere.